### DISCUSSION TITLE
Student developer pack not yet received

### DISCUSSION THREAD

**Discussion by montufood** [community/community#140351] [GitHub Education]
### Hi everyone,

i have been verified & approved for git hub education account but still haven't received Student developer pack  
please help me soon
thank you very much

**Comment by goldentalent626** [comment]
Even after approval, it can sometimes take a few days for the Student Developer Pack to be applied to your account. 
GitHub processes requests manually, which could lead to delays.
There are some other reasons for this delay.
For resolve this problem, you have to wait for a few days.
And if there is no answer, you can send message to github support team.

**Reply by montufood** [reply]
please provide me with github support team information, thank a lot!

**Reply by goldentalent626** [reply]
https://support.github.com/contact/education
Try this

**Comment by davevad93** [comment]
Hi @montufood ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @montufood was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
